### PR TITLE
Add call subcommand

### LINKED ISSUE #1: idea: add an extism plug-in system
@bhelx [brought up a great feature](https://discord.com/channels/1011124058408112148/1011124061100843020/1016095583380971602) in the Discord and I'm logging it here for posterity. 

> Is there a way to load a plugin into the runtime and invoke a function from the command line? I could how it might be a bit too early for that. But it would be useful when testing plugins.

Though maybe a bit limited, it's possible to add functionality here so that someone could run:

```sh
extism --load plugin.wasm --input '{"a": true}' call "is_it_true"

# or from stdin

echo '{"a": true}' | extism --load plugin.wasm call "is_it_true"
```

to test out their plugin in development.

### PR DESCRIPTION
**Pull request by zshipko**
Closes #1

It can be used like this:

```sh
$ extism call mycode.wasm function_name --input data
```

or 

```sh
$ echo data | extism call mycode.wasm function_name
```

### CODE REVIEW

File: extism_cli/__init__.py [hunk 1]
```diff
@@ -159,6 +159,12 @@ def asset_prefix(self, libc: str = "gnu"):
                   action="store_true",
                   help="Print link flags")
 
+# Call subcommand
+call = subparsers.add_parser("call")
+call.add_argument("wasm", help='WASM file')
+call.add_argument("--input", default=None, help='Plugin input')
+call.add_argument("function", help='Function name')
```
Comment (bhelx) [inline extism_cli/__init__.py:166]: Could maybe use some flags to set the debug level? or is that set another way?

Comment (zshipko) [reply extism_cli/__init__.py:166]: You read my mind! I just pushed a commit to do this

**Comment by nilslice** [comment]
getting this error after `pip3 install .` from the root and running the call cmd:

```sh
extism call --log-level debug ../extism/wasm/code.wasm count_vowels --input testing123
Traceback (most recent call last):
  File "/opt/homebrew/bin/extism", line 8, in <module>
    sys.exit(main())
  File "/opt/homebrew/lib/python3.10/site-packages/extism_cli/__init__.py", line 513, in main
    extism.set_log_level("-", args.log_level)
AttributeError: module 'extism' has no attribute 'set_log_level'
```

tried `--log-level` in different positions too.. I must be doing something wrong

**Comment by zshipko** [comment]
@nilslice - thanks, fix it - should have been `set_log_file`

**Pull request merged**